docs/querying/aggregations.md

```diff
@@ -126,7 +126,9 @@ Computes and stores the sum of values as 32-bit floating point value. Similar to
 
 ### `doubleMean` aggregator
 
-Computes and returns arithmetic mean of a column values as 64 bit float value. This is a query time aggregator only and should not be used during indexing.
+Computes and returns arithmetic mean of a column values as 64 bit float value. `doubleMean` is a query time aggregator only. It is not available for indexing.
+
+To accomplish mean aggregation on ingestion, refer to the [Quantiles aggregator](https://druid.apache.org/docs/latest/development/extensions-core/datasketches-quantiles.html#aggregator) from the DataSketches extension.
```

Comment:
The additional sentence looks good. Just to make this link more robust, we should use the following format:
  
```suggestion
To accomplish mean aggregation on ingestion, refer to the [Quantiles aggregator](../development/extensions-core/datasketches-quantiles.md#aggregator) from the DataSketches extension.
```